i18n/en.json

```diff
@@ -1,28 +1,28 @@
 {
-  "svgtranslate": "SVG Translate",
-  "info": "Help translate text labels on SVG files. Pick a file on commons to get started.",
   "beta": "This feature is in early testing.",
-  "learn-more": "Learn more & leave feedback!",
-  "powered-by-toolforge": "Powered by Toolforge",
-  "report-a-bug": "Report a bug",
-  "privacy-policy": "Privacy policy",
-  "view-on-github": "View on Github",
-  "licenced-under": "Licenced under $1",
-  "gpl3": "GPL 3.0 or later",
-  "made-by": "Made by the Wikimedia Foundation's $1",
   "commtech": "Community Tech team",
+  "gpl3": "GPL 3.0 or later",
   "help": "Help",
-  "pick-an-image": "Pick an image",
-  "pick-an-image-more": "from Commons.",
-  "translate-labels": "Translate",
-  "translate-labels-more": "the labels in the image.",
-  "update-commons": "Update Commons",
-  "update-commons-more": "or download directly to your computer.",
+  "info": "Help translate text labels on SVG files. Pick a file on commons to get started.",
+  "language-settings": "Language settings",
+  "learn-more": "Learn more & leave feedback!",
+  "licenced-under": "Licenced under $1",
   "login": "Log in",
   "logout": "Log out",
-  "language-settings": "Language settings",
+  "made-by": "Made by the Wikimedia Foundation's $1",
```

Comment:
I understand it wasn't introduced in this patch, however this is literally not translatable to Russian among a bunch of other languages because of cases. Patchwork sentences just never work.

Reply:
I'll change it (in a separate patch). I suspect something similar applies to the three tutorial messages too (`pick-an-image` etc.) where the first part of the phrase is in a larger font and bold. Do they need changing too?

Reply:
"Lego" messages are bad, but mainly only if they are re-used, no? It doesn't matter if the "Community Tech" message is uniquely for `made-by`, as would be described in qqq? The idea being to avoid HTML markup https://www.mediawiki.org/wiki/Localisation#Avoid_untranslated_HTML_markup_in_messages

It's different for MediaWiki where we use wikitext. Here we have to render the message as raw, so they can put in malicious or accidentally broken markup that breaks the display.